components/BlurhashImage.vue

```diff
@@ -58,14 +57,75 @@ export default Vue.extend({
       image: ''
     };
   },
+  computed: {
+    getImageHash(): string {
+      if (
+        this.imageType === ImageType.Primary &&
```

Comment:
We could probably make this a bit cleaner by using a switch on ImageType, like this:

```typescript
switch (this.imageType) {
  case ImageType.Primary:
    break;
  case ImageType.Thumb:
    break;
  // etc
}
```
It makes the following conditions a bit easier on the eyes, since they're now broken up.

Reply:
Good Idea, Updated